tidb/src/tidb/client.clj

```diff
@@ -0,0 +1,156 @@
+(ns tidb.client
+  (:require [clojure.string :as str]
+            [jepsen
+              [client :as client]
+              [generator :as gen]
+              [util :refer [timeout]]
+            ]
+            [clojure.java.jdbc :as j]
+  )
+)
+
+(defn conn-spec
+  "jdbc connection spec for a node."
+  [node]
+  {:classname   "org.mysql.jdbc.Driver"
+   :subprotocol "mysql"
+   :subname     (str "//" (name node) ":4000/jepsen")
```

Comment:
we don't create jepsen database and jepsen user by default. Seem you should use test database and root user directly here.